app/views/content/non-uk-teachers/visas-for-non-uk-trainees.md

```diff
@@ -55,48 +55,67 @@ keywords:
+
+* [Afghan Citizens Resettlement or Afghan Relocations and Assistance Policy scheme](https://www.gov.uk/government/publications/afghanistan-resettlement-and-immigration-policy-statement/afghanistan-resettlement-and-immigration-policy-statement-accessible-version)  
+* [British National (Overseas) visa](https://www.gov.uk/british-national-overseas-bno-visa)  
+* [Family visa](https://www.gov.uk/uk-family-visa)  
+* [Graduate visa](https://www.gov.uk/graduate-visa)  
+* [High Potential Individual visa](https://www.gov.uk/high-potential-individual-visa)  
+* [Homes for Ukraine Sponsorship Scheme](https://www.gov.uk/guidance/apply-for-a-visa-under-the-ukraine-sponsorship-scheme)  
+* [India Young Professionals Scheme visa](https://www.gov.uk/india-young-professionals-scheme-visa)  
+* [UK Ancestry visa](https://www.gov.uk/ancestry-visa)  
+* [Ukraine Extension Scheme](https://www.gov.uk/guidance/apply-to-stay-in-the-uk-under-the-ukraine-extension-scheme)  
+* [Ukraine Family Scheme visa](https://www.gov.uk/guidance/apply-for-a-ukraine-family-scheme-visa)  
+* [Youth Mobility Scheme visa](https://www.gov.uk/youth-mobility)
 
 $ukraine-visas$
 
-### Student visa 
+## Student visa for fee-paying training courses
 
-Most teacher training courses in England are fee-paying. You'll need a Student visa if you train on a fee-paying course. Your Student visa must be sponsored by your teacher training provider. 
+Most teacher training courses in England are fee-paying. This means you will need to pay tuition fees to train.
 
-### Skilled Worker visa 
+You will need a Student visa if you’re on a fee-paying course. Your Student visa must be sponsored by your teacher training provider. 
 
-Some teacher training courses in England are salaried – this means you [work and earn a salary while you train](/funding-and-support/salaried-teacher-training). Currently, places on these courses are very limited. You'll need a Skilled Worker visa if you train on a salaried training course. Your Skilled Worker visa must be sponsored by the school employing you.
+[Find out more about studying in the UK on a Student visa](https://www.gov.uk/student-visa).
 
-$refugees-asylum-seekers$
+### Bursaries and scholarships if you are on a Student visa 
+
+If you are training to teach physics or languages, you may be [eligible for a bursary or scholarship](/non-uk-teachers/fees-and-funding-for-non-uk-trainees).  
 
-### What you need to do to apply for your visa 
+If you successfully applied for a bursary or scholarship, you can use this in your Student visa application as proof that you have enough money to support yourself. 
 
-[Search for postgraduate teacher training](https://find-teacher-training-courses.service.gov.uk/) and filter your results by visa sponsorship. 
+Bursaries and scholarship payments are usually made by late October but you should confirm payment dates with your chosen provider.
```

Comment:
I think this is misleading - they are not lump sum payments - they are both paid over a minimum of 10 months? (Unless it is different for non-UK citizens - but I don't think so?)